modules/net-vpc-firewall/variables.tf

```diff
@@ -14,44 +14,16 @@
  * limitations under the License.
  */
 
-variable "network" {
-  description = "Name of the network this set of firewall rules applies to."
-  type        = string
-}
-
-variable "project_id" {
-  description = "Project id of the project that holds the network."
-  type        = string
-}
-
-variable "admin_ranges_enabled" {
-  description = "Enable admin ranges-based rules."
-  type        = bool
-  default     = false
-}
-
 variable "admin_ranges" {
   description = "IP CIDR ranges that have complete access to all subnets."
   type        = list(string)
   default     = []
 }
 
-variable "ssh_source_ranges" {
-  description = "List of IP CIDR ranges for tag-based SSH rule, defaults to the IAP forwarders range."
-  type        = list(string)
-  default     = ["35.235.240.0/20"]
-}
-
-variable "http_source_ranges" {
-  description = "List of IP CIDR ranges for tag-based HTTP rule, defaults to the health checkers ranges."
-  type        = list(string)
-  default     = ["35.191.0.0/16", "130.211.0.0/22", "209.85.152.0/22", "209.85.204.0/22"]
-}
-
-variable "https_source_ranges" {
-  description = "List of IP CIDR ranges for tag-based HTTPS rule, defaults to the health checkers ranges."
-  type        = list(string)
-  default     = ["35.191.0.0/16", "130.211.0.0/22", "209.85.152.0/22", "209.85.204.0/22"]
+variable "admin_ranges_enabled" {
```

Comment:
Can we remove this and use an empty list to prevent creation?

Reply:
We're already doing that

Reply:
then why do we still have that variable? :)

Reply:
no idea, backwards compatibility ;)